src/components/Reports/FourteenMonthReports/FourteenMonthReport.test.tsx

```diff
@@ -175,7 +229,7 @@ const TestComponent: React.FC<TestComponentProps> = ({
   currencyType,
   isNavListOpen = true,
   designationAccounts,
-  mutationSpy,
+  isSalary,
```

Comment:
Can we remove this prop and use `currencyType === FourteenMonthReportCurrencyType.Salary`?